jenkins_jobs/build_host_os/archive.sh

```diff
@@ -0,0 +1,16 @@
+JENKINS_MASTER=$(basename $JENKINS_URL)
+JENKINS_BUILD_ARCHIVE_DIR_NAME="archive"
+JENKINS_BUILD_ARCHIVE_URL=${JENKINS_MASTER}:${JENKINS_HOME}/jobs/${JOB_NAME}/builds/${BUILD_NUMBER}/${JENKINS_BUILD_ARCHIVE_DIR_NAME}
+
+alias rsync_upload="rsync -e \
+    'ssh -i ${HOME}/.ssh/jenkins_id_rsa' \
+    --verbose --compress --stats --times --perms"
```

Comment:
I think using a function instead of an alias would improve legibility

```
rsync_upload() {
    rsync -e 'ssh -i ${HOME}/.ssh/jenkins_id_rsa' \
              --verbose --compress --stats --times --perms \
              $@ $JENKINS_BUILD_ARCHIVE_URL
}

rsync_upload --recursive $JENKINS_BUILD_ARCHIVE_DIR_NAME
rsync_upload SUCCESS
rsync_upload BUILD_TIMESTAMP
rsync_upload --recursive repository
```